executor/ddl_test.go

```diff
@@ -90,7 +91,7 @@ func (s *testSuite3) TestCreateTable(c *C) {
 	chk := rs.NewChunk()
 	it := chunk.NewIterator4Chunk(chk)
 	for {
-		err1 := rs.Next(ctx, chk)
+		err1 := rs.Next(ctx, &execution.ExecRequest{RetChunk: chk})
```

Comment:
Suggest to use a method like `NewExecRequest(chk)` to create this struct.
If someone add a new field into `ExecRequest` with a specific default value, then he have to modify all places again.

Reply:
I prefer 0 or false a default value, and this make more feeling like-named parameter, but add ctor is ok too... ok, let's use change